src/commands/Minion/chop.ts

```diff
@@ -75,6 +75,11 @@ export default class extends BotCommand {
 			throw `${msg.author.minionName} needs ${log.level} Woodcutting to chop ${log.name}.`;
 		}
 
+		const QP = msg.author.settings.get(UserSettings.QP);
+		if (QP < 25 && log.name === 'Sulliusceps') {
+			throw `You need 25 QP to visit fossil island and cut Sulliusceps`;
+		}
```

Comment:
this isnt a good way of doing it..at best it should use the ID, or better it should have `qpRequired: 25` in the log object